src/index.html

```diff
@@ -6,9 +6,31 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="#home"><img src="images/logo.png" alt="logo"></a>
+
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item"><a class="is-active nav-link" href="#Apple">Apple</a></li>
+          <li class="nav-item"><a class="nav-link" href="#Samsung">Samsung</a></li>
+          <li class="nav-item"><a class="nav-link" href="#Smartphones">Smartphones</a></li>
```

Comment:
```suggestion
          <li class="nav-item">
            <a class="nav-link" href="#Samsung">Samsung</a>
          </li>
          
          <li class="nav-item">
            <a class="nav-link" href="#Smartphones">Smartphones</a>
          </li>
```

Add empty line between multiple sibling blocks of html